packages/circuit-ui/components/TopNavigation/TopNavigation.tsx

```diff
@@ -52,6 +53,15 @@ export interface TopNavigationProps
    */
   user?: UserProps;
   isLoading?: boolean;
+  /**
+   * link to page main content
+   * to comply with WCAG 2.1 https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks.html
```

Comment:
I would add a bit more context to these doc comments. And make sure to capitalize the first letter :) 

```suggestion
   * Hash link to the page's main content to enable keyboard and screen reader
   * users to skip over the navigation links. Required to comply with 
   * [WCAG 2.1 SC 2.4.1](https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks.html)
```